storm/ci/storm.rake

```diff
@@ -0,0 +1,60 @@
+require 'ci/common'
+
+def storm_version
+  ENV['FLAVOR_VERSION'] || 'latest'
+end
+
+def storm_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/storm_#{storm_version}"
+end
+
+namespace :ci do
+  namespace :storm do |flavor|
+    task before_install: ['ci:common:before_install']
+
+    task install: ['ci:common:install'] do
+      # sample docker usage
+      # sh %(docker create -p XXX:YYY --name storm source/storm:storm_version)
```

Comment:
Could we run the docker here ? Will the test pass in the CI if we don't ?

Reply:
Yes, was working on that last night, still proving out the mini-cluster setup + build the topology and deploy it, then writing a dedicated integration check for ci-only.  Will update the README for contributors as well.  Does our CI support docker compose? would simplify into a single command and make this more local developer friendly to avoid drift.

Reply:
Docker compose should work (it's used in the varnish CI for integration-core)